src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { TodoItem } from '../TodoItem';
+import { useTodos } from '../../TodosContext';
+
+export const TodoList: React.FC = () => {
+  const { filteredTodos, tempTodos } = useTodos();
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <TransitionGroup>
+        {filteredTodos.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
+            <TodoItem todo={todo} key={todo.id} />
+          </CSSTransition>
+        ))}
+
+        {tempTodos[0]?.id === 0 && (
```

Comment:
```suggestion
        {!tempTodos[0]?.id && (
```

Reply:
I can't change this condition. In this case, i am comparing ids. Todo id should be equal to zero.